packages/common/src/ui/components/inputs/Autocomplete/AutocompleteList.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
```

Comment:
styling for this one ended up being pretty annoying. Potentially would have been better off just using the `useAutocomplete` hook from mui. Still needs polish but I got a bit bored of styling so will revisit ..